### ISSUE TITLE
Performance improvements for large log entries

### ISSUE THREAD

**Issue by tj** [apex/roadmap#68]
Two customers have mentioned now that the listing becomes unresponsive for them due to a large number or deeply nested fields. This might need some profiling but the first two solutions which come to mind for me are:

- [ ] virtual scrolling so only the on-screen logs are in the DOM
- [ ] truncate the fields, currently they're all displayed

Ideal truncation is a bit subjective, if you omit important fields it could be annoying but I'm not sure there's away around it, the DOM can only do so much :D

**Comment by AndyOsei** [comment]
what about providing a means to lazy load fields so the user can expand and view them on-demand..?

**Comment by tj** [comment]
Yeah I like that, just a ellipsis or something you click sounds decent to me!

I think it mightttt need some kind of way to select which fields are always visible, it could get annoying constantly expanding if you need one which is truncated. 

It'd also maybe be nice to always include the fields in the query, should be easy enough to grab those.

**Comment by AndyOsei** [comment]
I agree.
